<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Input_score</class>
 <widget class="QMainWindow" name="Input_score">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="1">
     <widget class="QTabWidget" name="input_tab">
      <property name="minimumSize">
       <size>
        <width>782</width>
        <height>539</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-color: rgb(252, 253, 255);</string>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="iconSize">
       <size>
        <width>13</width>
        <height>13</height>
       </size>
      </property>
      <widget class="QWidget" name="single_input">
       <attribute name="title">
        <string>单个录入</string>
       </attribute>
       <widget class="QWidget" name="widget" native="true">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>100</y>
          <width>251</width>
          <height>281</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">
color: rgb(0, 0,0);</string>
        </property>
        <layout class="QFormLayout" name="formLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="input_Chinese_label">
           <property name="styleSheet">
            <string notr="true">font: 16pt &quot;宋体&quot;;</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;语文：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="input_Chinese_line">
           <property name="styleSheet">
            <string notr="true">border:1px groove gray;border-radius:
7px;padding:2px 4px;</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="input_math_label">
           <property name="styleSheet">
            <string notr="true">font: 16pt &quot;宋体&quot;;</string>
           </property>
           <property name="text">
            <string>数学：</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="input_math_line">
           <property name="styleSheet">
            <string notr="true">border:1px groove gray;border-radius:
7px;padding:2px 4px;</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="input_English_label">
           <property name="styleSheet">
            <string notr="true">font: 16pt &quot;宋体&quot;;</string>
           </property>
           <property name="text">
            <string>英语：</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="input_English_line">
           <property name="styleSheet">
            <string notr="true">border:1px groove gray;border-radius:
7px;padding:2px 4px;</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="input_physics_label">
           <property name="styleSheet">
            <string notr="true">font: 16pt &quot;宋体&quot;;</string>
           </property>
           <property name="text">
            <string>物理：</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="input_physics_line">
           <property name="styleSheet">
            <string notr="true">border:1px groove gray;border-radius:
7px;padding:2px 4px;</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="input_chemistry_label">
           <property name="styleSheet">
            <string notr="true">font: 16pt &quot;宋体&quot;;</string>
           </property>
           <property name="text">
            <string>化学：</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QLineEdit" name="input_chemistry_line">
           <property name="styleSheet">
            <string notr="true">border:1px groove gray;border-radius:
7px;padding:2px 4px;</string>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="input_biology_label">
           <property name="styleSheet">
            <string notr="true">font: 16pt &quot;宋体&quot;;</string>
           </property>
           <property name="text">
            <string>生物：</string>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="QLineEdit" name="input_biology_line">
           <property name="styleSheet">
            <string notr="true">border:1px groove gray;border-radius:
7px;padding:2px 4px;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>110</y>
          <width>251</width>
          <height>101</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">
color: rgb(0, 0, 0);</string>
        </property>
        <layout class="QFormLayout" name="formLayout_3">
         <item row="2" column="1">
          <widget class="QLineEdit" name="input_name_line">
           <property name="styleSheet">
            <string notr="true">border:1px groove gray;border-radius:
7px;padding:2px 4px;</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="input_name_label">
           <property name="styleSheet">
            <string notr="true">font: 16pt &quot;宋体&quot;;</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;姓名：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="input_num_line">
           <property name="styleSheet">
            <string notr="true">border:1px groove gray;border-radius:
7px;padding:2px 4px;</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="input_num_label">
           <property name="styleSheet">
            <string notr="true">font: 16pt &quot;宋体&quot;;</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;学号：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <spacer name="verticalSpacer_6">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="input_single_confirm">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>420</y>
          <width>111</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 16pt &quot;宋体&quot;;
background-color: rgb(218, 205, 255);</string>
        </property>
        <property name="text">
         <string>确定导入</string>
        </property>
       </widget>
       <widget class="QPushButton" name="input_single_return">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>420</y>
          <width>111</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 16pt &quot;宋体&quot;;
background-color: rgb(204, 204, 191);</string>
        </property>
        <property name="text">
         <string>返回</string>
        </property>
       </widget>
       <widget class="QLabel" name="revise_title_2">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>10</y>
          <width>141</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>录入成绩</string>
        </property>
       </widget>
       <zorder>widget</zorder>
       <zorder>input_single_confirm</zorder>
       <zorder>input_single_return</zorder>
       <zorder>widget_2</zorder>
       <zorder>revise_title_2</zorder>
      </widget>
      <widget class="QWidget" name="excel_input">
       <attribute name="title">
        <string>EXCEL导入</string>
       </attribute>
       <widget class="QPushButton" name="input_excel_confirm">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>300</y>
          <width>111</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(218, 205, 255);
font: 16pt &quot;宋体&quot;;</string>
        </property>
        <property name="text">
         <string>确认导入</string>
        </property>
       </widget>
       <widget class="QPushButton" name="input_excel_return">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>300</y>
          <width>111</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 16pt &quot;宋体&quot;;
background-color: rgb(204, 204, 191);</string>
        </property>
        <property name="text">
         <string>返回</string>
        </property>
       </widget>
       <widget class="QLabel" name="revise_title">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>10</y>
          <width>141</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>录入成绩</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
